app/graphql/queries/case_study.graphql

```diff
@@ -778,6 +778,17 @@ query case_study($base_path: String!) {
           content_id
           description
           document_type
+          links {
+            world_location_news {
+              analytics_identifier
+              base_path
+              content_id
+              document_type
+              locale
+              title
+              description
```

Comment:
Just a little nitpick: could these be sorted into alphabetical order, so it's easier to see what's being included at a glance.